microsoft-edge/devtools-guide-chromium/sensors/index.md

```diff
@@ -0,0 +1,230 @@
+
+To emulate idle states:
+
+1. [Open the Sensors tool](#open-the-sensors-tool).  For this tutorial, you can try it on the [Ephemeral Canvas](https://idle-detection.glitch.me/) demo page.
+
+1. In the demo page, select the **Ephemeral** checkbox, and then click the **Allow** button to grant the demo page the idle detection permission.
+
+   <!-- ![Granting the idle detection permission on a demo page](./index-images/granting-idle-detection.png) can't get dialog again -->
+
+1. Reload the page.
+
+1. From the **Emulate Idle Detector state** drop-down, select one of the following:
+   * **No idle emulation**
+   * **User active, screen unlocked**
+   * **User active, screen locked**
+   * **User idle, screen unlocked**
+   * **User idle, screen locked**
+
+   ![Selecting an idle and locked state on a demo page](./index-images/idle-detection-demo.png)
+
+In this example, DevTools emulates a **User idle, screen locked** state and, in this case, the demo page starts the 60 second countdown to clear the canvas.
+
+
+<!-- ====================================================================== -->
+## Emulate hardware concurrency
+<!-- https://developer.chrome.com/docs/devtools/sensors#hardware-concurrency -->
+
+To emulate how your website performs on devices with different numbers of processor cores, you can override the value reported by the [navigator.hardwareConcurrency](https://developer.mozilla.org/docs/Web/API/Navigator/hardwareConcurrency) property.  Some applications use this property to control the degree of parallelism of their application, for example, to control [Emscripten `pthread`](https://emscripten.org/docs/porting/pthreads.html) pool size.
+
+To emulate hardware concurrency:
+
+1. [Open the Sensors tool](#open-the-sensors-tool).
+
+1. Near the bottom, select the **Hardware concurrency** checkbox.
+
+1. In the number input box, enter the number of cores you want to emulate:
+
+   ![Turned on 'Hardware concurrency' with the number of cores set to 10](./index-images/hardware-concurrency.png)
+
+   Caution: Exceeding the default value may degrade performance.
+
+To revert to the default value, click the **Reset** (![Reset icon](./index-images/reset-icon.png)) button.
+
+
+<!-- ====================================================================== -->
+## Emulate CPU pressure
+<!-- https://developer.chrome.com/docs/devtools/sensors#cpu-pressure -->
+
+[CPU or compute pressure](https://developer.chrome.com/docs/web-platform/compute-pressure) is a set of reported states that give you an idea of how much workload the system's compute is undergoing and whether it's near to the limit or not.  This information lets you adapt your real-time application, for example, video conferencing or video game, to take advantage of all the available processing power while keeping the system responsive by adapting the workloads in real time.
```

Comment:
I would remove the link to developer.chrome.com here. There's already a link to https://developer.mozilla.org/en-US/docs/Web/API/Compute_Pressure_API in the next paragraph, and I think that's enough.